<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="mentionLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>请输入一个小于1000的正整数</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="insertButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mentionLabel_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>温馨提示：树层数最多为5</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除全树</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BSTButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>普通二叉搜索树</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AVLButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>平衡二叉树</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NLRButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>前序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LNRButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>中序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LRNButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>后序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LevelButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>层序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RBTButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>红黑树</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>随机生成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
